trigger-actions-framework/main/default/classes/TriggerActionFlow.cls

```diff
@@ -182,7 +173,10 @@ public inherited sharing class TriggerActionFlow implements TriggerAction.Before
 	}
 
 	private Invocable.Action getAction() {
-		return Invocable.Action.createCustomAction(FLOW_STRING, flowName);
+		return Invocable.Action.createCustomAction(
+			TriggerActionFlowBypassProcessor.FLOW_STRING,
```

Comment:
nit: It feels _slightly_ odd that `TriggerActionFlow` depends upon a constant defined in the `TriggerActionFlowBypassProcessor`. 

Do you think it would be beneficial for the reader if we moved these to a new `TriggerActionConstants` class?

Reply:
@mitchspano yes, absolutely. I was going for minimum friction and new introductions with this change; I already felt it was "out of character" for each of the dependent classes to be reaching into `TriggerActionFlow`. I'm happy to make this change, or you're welcome to!

Reply:
Resolved with the introduction of `TriggerActionConstants`